auth-web/src/components/auth/staff/StaffAccountManagement.vue

```diff
@@ -147,6 +147,10 @@ export default class StaffAccountManagement extends Vue {
     await this.syncPendingInvitationOrgs()
   }
 
+  private get pagesEnum () {
```

Comment:
Pages are static right? we should avoid the computed get for that and use normal variable.